fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java

```diff
@@ -74,19 +74,23 @@
  *    "aggregations": [
  *      {
  *        "name": "resultSketches",
- *        "fieldName": "rawSketches",
+ *        "fieldName": "sketchesPerRow",
  *        "type": "longSum"
  *      },
```

Comment:
Why is this here if its just overriden in the postAgg section?

Reply:
The fieldname is the downward pointing reference, so resultSketches is the total of the sketchesPerRow from the inner layer.

Reply:
ah, I missed that I was looking at fieldname rather than just `name`. makes sense now.